src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -307,18 +327,48 @@ class CrossDeviceLinkUI extends Component {
     this.viewOptionBtn.blur()
   }
 
+  getRequiredViewRenders = () => {
+    const { _crossDeviceLinkMethods } = this.props
+    const defaultViewRendersMap = {
+      qr_code: this.renderQrCodeSection,
+      sms: this.renderSmsLinkSection,
+      copy_link: this.renderCopyLinkSection,
+    }
+    if (
+      !_crossDeviceLinkMethods ||
+      (_crossDeviceLinkMethods && _crossDeviceLinkMethods.length < 1)
+    ) {
+      return defaultViewRendersMap
+    }
+    if (
+      _crossDeviceLinkMethods &&
+      configHasInvalidViewIds(_crossDeviceLinkMethods)
+    ) {
+      return defaultViewRendersMap
+    }
```

Comment:
💅🏻 [nitpick]

Could simplified to 2 expressions (or 3 if you count the optional chaining operator) if we take advantage of the fact an undefined array and an empty one are essentially the same fail condition.

```suggestion
    if (
      !_crossDeviceLinkMethods?.length ||
      configHasInvalidViewIds(_crossDeviceLinkMethods)
    ) {
      return defaultViewRendersMap
    }
```